cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis.hs

```diff
@@ -660,6 +661,75 @@ runGenesisCreateStakedCmd
 
 -- -------------------------------------------------------------------------------------------------
 
+updateOutputTemplate
```

Comment:
Why this code is duplicated: https://github.com/IntersectMBO/cardano-cli/pull/581/files#diff-8cad416779bd68d2dc00788d194d042bec6fc12b8231d09f6169b911ab37c1a7L584 ?

Reply:
@carbolymer> It's not duplicated: the code you refer to was previously shared was previously used both by `create-staked` and `create-testnet-data`. To keep `create-staked`'s existing behavior, I unshared it, by copying the preexisting version to `create-staked`'s file.

Reply:
There's a lot of similar code, like utilities in `where`. Especially this funds distribution part:
```haskell
                                distribute (nonDelegCoin - subtractForTreasury) nUtxoAddrsNonDeleg  utxoAddrsNonDeleg
                                ++
                                distribute (delegCoin - subtractForTreasury)    nUtxoAddrsDeleg     utxoAddrsDeleg
                                ++
                                mkStuffedUtxo stuffedUtxoAddrs
```
Can you de-duplicate parts of this function?

Reply:
Since we don't want to change behavior of `create-staked` and don't plan on improving it, I am more in favor of duplicating, so that we have ours hand more free for changing `create-testnet-data` in the future; if you don't mind.

Reply:
I'm in favour of de-duplicating. I'm un-requesting changes. @Jimbo4350 your call.

Reply:
Let's focus on `create-testnet-data` for now. Benchmarking depends on `create-staked`. We can revisit in the future and clean it up. I would rename it to something create testnet data specific though.

Reply:
This function is not exported by the module, so I think its name doesn't need to be `create-testnet-data` specific.